Documentation/cross-building.md

```diff
@@ -0,0 +1,76 @@
+Cross Compilation for ARM on Linux
+==================================
+
+Through cross compilation, on Linux it is possible to build CoreCLR for arm or arm64.
+
+Requirements
+------------
+
+You need a Debian based host and the following packages needs to be installed:
+
+    $ sudo apt-get install qemu qemu-user-static binfmt-support debootstrap
+
+In addition, to cross compile CoreCLR the binutils for the target are required. So for arm you need:
+
```

Comment:
Do these steps differ from CoreCLR repo? If not, it may be better to just link to the steps in CoreCLR repo, until they start to differ. If they do differ already, all references to CoreCLR should be replace with CoreRT in the version for CoreRT repo.

If yet,